lib/system/endb.nim

```diff
@@ -76,29 +76,52 @@ proc `==`(a, b: StaticStr): bool =
 proc `==`(a: StaticStr, b: cstring): bool =
   result = c_strcmp(unsafeAddr a.data, b) == 0
 
-proc write(f: File, s: StaticStr) =
+proc write(f: CFileStar, s: cstring) = c_fputs(s, f)
+proc writeLine(f: CFileStar, s: cstring) =
+  c_fputs(s, f)
+  c_fputs("\n", f)
+
+proc write(f: CFileStar, s: StaticStr) =
   write(f, cstring(unsafeAddr s.data))
 
+proc write(f: CFileStar, i: int) =
+  when sizeof(int) == 8:
+    discard c_fprintf(f, "%lld", i)
+  else:
+    discard c_fprintf(f, "%ld", i)
+
+proc close(f: CFileStar): cint {.
```

Comment:
- [ ] why not follow existing convention, ie `c_fclose` (which was previous name in sysio.nim)
- [ ] like you said yourself, `discardable` is evil; it also makes errors silently accepted

for reference here was the previous definition:
```nim
proc c_fclose(f: File): cint {.
  importc: "fclose", header: "<stdio.h>".}
```

Reply:
it's ENDB, I hacked it together.